docs/console/project-configuration/create-a-project.mdx

```diff
@@ -92,28 +92,24 @@ The Project creation process is divided into three phases:
       If the path does not exist, the Project will not be created.
 
       For example, if your organization is reachable at the URL **`https://github.com/mia-platform`**, <br/>
-      then the `basePath` must be **`mia-platform`**
+      then the Git repo path must be **`mia-platform`**
 
     </TabItem>
 
     <TabItem value="azure-devops" label="Azure DevOps">
```

Comment:
The configuration we had was the one for the default git provider in the company, I think it was in the wrong place. We should move it in the `console/company-configuration/providers/management#set-a-default-provider` paragraph, instead of deleting it. @fredmaggiowski correct me if I'm wrong.